lib/keeper/datastore.rb

```diff
@@ -2,25 +2,37 @@ module Keeper
   module Datastore
     class << self
       # @!visibility private
-      def expire(token, time)
-        set_with_expiry(token, time)
+      def rotate(jti, seconds)
```

Comment:
In the original version that you authored it was called `token` because the entire token was being passed. Since then it was updated to only store the token's uuid, which per the rfc is known as `jti` for json token id. This naming change is to reflect that functional change.